### PR TITLE
[FIX] stock: make the location current stock view non-editable

### PR DESCRIPTION
**Pull request by DjamelTouati**
Steps to reproduce the bug:
- Enable “Storage Locations” option in the inventory settings
- Go to Inventory > Configuration > Warehouse > locations
- Choose any Location > Click on Current stock > Select any product
- Try to add two line

Problem:
Traceback is triggered, Because the line should not be added twice(like in “Inventory Adjustments” view)
and as in the `_onchange_location_or_product_id` we access the quantity,
it should be only one quant: https://github.com/odoo/odoo/blob/15.0/addons/stock/models/stock_quant.py#L517

In v14, this list view is not editable, but since the refactor in v15 added the new view `stock.quant.inventory.tree.editable`,
it is being called by this action

Solution:
set the `view_id` to `stock.quant.tree`, which is the view it was previously using

opw-2779795

https://user-images.githubusercontent.com/78867936/158364388-ca8a3837-26ee-4ef1-a04a-bcbc6c0d8e95.mp4

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/views/stock_quant_views.xml [hunk 1]
```diff
@@ -180,12 +180,6 @@
         <field name="context">{'search_default_internal_loc': 1, 'search_default_productgroup':1, 'search_default_locationgroup':1}</field>
         <field name="res_model">stock.quant</field>
     </record>
-    <record model="ir.actions.act_window" id="location_open_quants"> <!-- Used in location -->
-        <field name="context">{'search_default_productgroup': 1}</field>
-        <field name="domain">[('location_id', 'child_of', active_ids)]</field>
-        <field name="name">Current Stock</field>
-        <field name="res_model">stock.quant</field>
-    </record>
```
Comment (ticodoo) [inline addons/stock/views/stock_quant_views.xml:188]: I think maybe you can do a simple one line solution since in v14 this list view is **not** editable :thinking:. I think since the refactor in v15 added the new view `stock.quant.inventory.tree.editable`, it is being called by this action. You should be able to set its `view_id`  to `stock.quant.tree`, which is the view it was previously using. If the customer really wants to be able to edit products qtys by location, they can still do so using the Inventory Adjustments grouped/filtered by Location or with Reporting > Inventory Report.

TLDR: we can probably minimize the stable changes since the user can still do the same word in other views.

File: addons/stock/views/stock_quant_views.xml [hunk 2]
```diff
@@ -180,13 +180,21 @@
...
         <field name="context">{'search_default_productgroup': 1}</field>
         <field name="domain">[('location_id', 'child_of', active_ids)]</field>
         <field name="name">Current Stock</field>
         <field name="res_model">stock.quant</field>
     </record>
 
+    <record id="location_open_quants_tree_view" model="ir.actions.act_window.view">
+        <field name="sequence" eval="1"/>
+        <field name="view_mode">tree</field>
+        <field name="view_id" ref="view_stock_quant_tree"/>
```
Comment (ticodoo) [inline addons/stock/views/stock_quant_views.xml:194]: No need for a new action, you can add the `view_id` directly to the `location_open_quants` action :wink:

Comment (ticodoo) [inline addons/stock/views/stock_quant_views.xml:194]: No need for a new action, you can add the `view_id` directly to the `location_open_quants` action :wink:

**Comment by DjamelTouati** [comment]
Indeed, thanks for your help !

**Comment by ticodoo** [comment]
@robodoo r+
